packages/web3-utils/types/tslint.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Same here I think it should be possible to have this as a global config.

Reply:
as spoken offline dtslint do not support the extends property at the moment - I am going to do a PR for them so we can support this but for now i will mark this as resolved